visualization/app/codeCharta/ui/filePanel/filePanelDeltaSelector/filePanelDeltaSelector.component.html

```diff
@@ -3,8 +3,14 @@
 		{{ file.file.fileMeta.fileName | removeExtension }}
 	</mat-option>
 </mat-select>
-
-<div class="pictogram" [style.background]="pictogramBackground$ | async"></div>
+<button
+	class="direction-switch"
+	[disabled]="(comparisonFile$ | async) === undefined"
```

Comment:
I think we should add some styles for disabled. Maybe we can use matButton directive to use common styeles? In that case we could probably remove `background-color: theme.$cc-primary-color;` and hover its background-color
![image](https://user-images.githubusercontent.com/3596742/182796782-cae1d294-fc6c-45a1-a739-e8c04f9d57d6.png)